test/node/object_id_tests.js

```diff
@@ -6,16 +6,12 @@ const util = require('util');
 const ObjectId = BSON.ObjectId;
 
 describe('ObjectId', function () {
-  /**
-   * @ignore
-   */
   it('should correctly handle objectId timestamps', function (done) {
-    // var test_number = {id: ObjectI()};
-    var a = ObjectId.createFromTime(1);
+    let a = ObjectId.createFromTime(1);
     expect(Buffer.from([0, 0, 0, 1])).to.deep.equal(a.id.slice(0, 4));
     expect(1000).to.equal(a.getTimestamp().getTime());
 
-    var b = new ObjectId();
+    let b = new ObjectId();
```

Comment:
This and the declaration in L10 can be a `const`. One somewhat weird thing in JS that you might not be aware of is that `const` does not mean "unchanging", it only means "unchanging reference", so even though we set a property on `b` in L14, as far as JS is concerned, `b` is still a const, because its reference is unchanged.